dev/AppNotifications/AppNotificationBuilder/AppNotificationBuilderUtility.h

```diff
@@ -16,6 +20,24 @@ namespace AppNotificationBuilder
     using namespace winrt::Microsoft::Windows::AppNotifications::Builder;
 }
 
+inline std::unordered_map<wchar_t, std::wstring> GetXmlEscapeEncodings()
+{
+    static std::unordered_map<wchar_t, std::wstring> encodings = { { L'&', L"&amp;"}, { L'\"', L"&quot;"}, {L'<', L"&lt;"}, {L'>', L"&gt;"}, {L'\'', L"&apos;"}};
+    return encodings;
+}
+
+inline std::unordered_map<wchar_t, std::wstring> GetPercentEncodings()
+{
+    static std::unordered_map<wchar_t, std::wstring> encodings = {{ L'%', L"%25"}, {L';', L"%3B"}, {L'=', L"%3D"} };
```

Comment:
Just put this in the first map. Why do we need a second API?

Reply:
We need the second API because the only use for percent encoding is for arguments. %;= don't need any special encoding for anything else.